src/app/application/components/app-view/AppView.vue

```diff
@@ -148,30 +148,29 @@ onBeforeUnmount(() => {
 }
 .app-view-title-bar {
   h1, h2, h3, h4, h5, h6  {
-    color: var(--black-500);
```

Comment:
question: this was removed and not replace; just making sure it was intentional (if yes just resolve the comment)

Reply:
That was intentional.